webapp/portlet/src/main/webapp/common/js/SuggesterService.js

```diff
@@ -24,7 +24,8 @@ function searchSpaces(filter, items, excludeNonRedactor, noRedactorSpace) {
     .then(resp => resp && resp.ok && resp.json())
     .then(data => {
       data.spaces.forEach((item) => {
-        if ((!noRedactorSpace || !item.redactorsCount) && (!excludeNonRedactor || item.isRedactor || !item.redactorsCount)) {
+      // canEdit = true if manager or super manager
+        if ((!noRedactorSpace || !item.redactorsCount) && (!excludeNonRedactor || item.isRedactor || !item.redactorsCount) || (!excludeNonManager || item.canEdit)) {
```

Comment:
Why adding item.canEdit here ?

Reply:
it return true if user is manager or super manager